docs/paper/dev/api/pdc.md

```diff
@@ -86,17 +86,34 @@ The PDC supports a wide range of data types, such as:
   PersistentDataContainer newContainer = container.getAdapterContext().newPersistentDataContainer();
   ```
 - `Boolean`
+- `Lists`. Represent lists of data that can be stored via another persistent data type. You may create them via:
+  ```java
+  // Storing a list of strings in a container by verbosely creating
+  // a list data type wrapping the string data type.
+  container.set(
+    key,
+    PersistentDataType.LIST.listTypeFrom(PersistentDataType.STRING),
+    List.of("a", "list", "of", "strings")
+  );
+
+  // Storing a list of strings in a container by using the api
+  // provided pre-definitions of commonly used list types.
+  container.set(key, PersistentDataType.LIST.strings(), List.of("a", "list", "of", "strings"));
```

Comment:
Is it worth listing the inbuild ones? I know you can see them from the javadocs so I’m not sure here.

Reply:
That's a lot to list and kinda just mirrors the existing list. I wouldn't add it but if others think that is benefitial, sure

Reply:
Fairs, no issue.